packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.html

```diff
@@ -0,0 +1,102 @@
+<div>
+    <oui-back-button data-on-click="$ctrl.goBack()"
+        ><span data-translate="dedicatedCloud_manage_nsx_edge_title"></span>
+    </oui-back-button>
+</div>
+
+<p
+    class="my-4"
+    data-translate="dedicatedCloud_tab_datacenter_global_information"
+></p>
+
+<oui-message
+    data-ng-if="!$ctrl.hasScalingCapabilities"
+    class="d-block mb-4"
+    data-type="info"
+>
+    <span data-translate="dedicatedCloud_manage_nsx_missing_ressources"></span>
+
+    <a
+        class="oui-link oui-link_icon"
+        data-ng-href="{{:: $ctrl.getDatacenterLink() }}"
+        target="_blank"
+        rel="noopener"
+    >
+        <span data-translate="dedicatedCloud_nsx_link_datastore_order"></span>
+        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
+    </a>
+
+    <a
+        class="oui-link oui-link_icon"
+        data-ng-href="{{:: $ctrl.getHostLink() }}"
+        target="_blank"
+        rel="noopener"
+    >
+        <span data-translate="dedicatedCloud_nsx_link_hosts_order"></span>
+        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
+    </a>
+</oui-message>
+
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row d-md-flex mt-3">
```

Comment:
```suggestion
<div class="text-center" data-ng-if="$ctrl.loading">
    <oui-spinner></oui-spinner>
</div>

<div class="row d-md-flex mt-3" data-ng-if="!$ctrl.loading">
```
I don't kown if we have to display the rest of page if it's in loading

Reply:
We can : loading is used to display the spinner when resize is launched, not when the page is loading.